Java/cp/Defanged_IPv4.java

```diff
@@ -0,0 +1,39 @@
+/* Given a valid (IPv4) IP address, return a defanged version of of that IP address. A defanged IP address replaces every period "." with "[.]".
```

Comment:
```suggestion
/* Given a (IPv4) IP address, return a defanged version of of that IP address. A defanged IP address replaces every period "." with "[.]".
```
I have edited the problem description because if the given IP address is always valid then the validation part of the code would become redundant.

Reply:
Done @atarax665